frontend/src/pages/agreements/list/AgreementsTable.jsx

```diff
@@ -0,0 +1,34 @@
+import "./AgreementsList.scss";
+import { AgreementTableRow } from "./AgreementTableRow";
+
+/**
+ * Agreement table.
+ * @param {Object} props - The component props.
+ * @param {Object[]} props.agreements - Array of Agreement to display in the table.
+ * @returns {ReactNode} The rendered component.
+ */
+export const AgreementsTable = ({ agreements }) => {
+    return (
+        <table className="usa-table usa-table--borderless width-full ">
```

Comment:
wonder if we need like a fallback option if no agreements are found?

Reply:
Added this.

Reply:
Don't think we even need to render the table if there aren't any agreements. Maybe something like: 
```js
{
  agreements.length > 0 ?  (
    // render table
  ) : (
   // no agreements
  )
}
```